config.yaml

```diff
@@ -83,4 +83,23 @@ core_config_version: 0
 
 
 # (OPTIONAL | Default: ""). Used to prepend a base path to all APIs when querying the core.
-# base_path:
\ No newline at end of file
+# base_path:
+
+# (OPTIONAL | Default: "BCRYPT"). The password hashing algorithm to use. Values are "ARGON2" | "BCRYPT"
+# password_hashing_alg:
+
+# (OPTIONAL | Default: 11). Number of rounds to set for bcrypt password hashing
```

Comment:
I think this should be `bcrypt_log_rounds` or `bcrypt_cost`.

Reply:
sounds good.

Reply:
I have updated this to `bcrypt_log_rounds`.